stable/opa/values.yaml

```diff
@@ -45,7 +45,23 @@ admissionControllerKey: ""
 
 # Docker image and tag to deploy.
 image: openpolicyagent/opa
-imageTag: 0.10.1
+imageTag: 0.10.2
+
+mgmt:
+  enabled: true
+  image: openpolicyagent/kube-mgmt
+  imageTag: 0.8-dev
+  extraArgs: []
+  configmapPolicies:
+    enabled: false  
+    namespaces: [opa, kube-federation-scheduling-policy]
+    requireLabel: true
+  replicate:
+    cluster: []
```

Comment:
How does the user set these parameters?

```
helm install opa --set mgmt.replicate.namespace[0]=v1/namespaces \
   --set mgmt.replicate.namespace[1]=v1/pods
```

Like this?

Reply:
from the cli? Yeah. or I think you can do --set mgmt.replicate.namespace={v1/namespaces,v1/pods}

Reply:
@kfox1111 we should include a comment here to remind people to update the RBAC rules below otherwise kube-mgmt will get denied when it tries to list/watch.